.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_API_KEY=iSAbLNTnHlhG33VP27yEQDITzhxAZ1iZ
```

Comment:
I recommend that you add this file to  your .gitignore as this are api keys

Reply:
Done! Thnks

Reply:
Done. Thnks.

Reply:
You pushed it? I can still see your .env config